File: C/Basic Maths/Addition.c
#include <stdio.h>

// program to add two numbers
void main() {
    float num1, num2, sum;
    printf("Enter two numbers to add: ");
    scanf("%f %f", &num1, &num2);
    sum = num1 + num2;
    printf("Sum is: %.2f", sum);
}



//============================================================================
//============================================================================



// program to add n number of integers
void main() {
    int num, sum=0, i, n;
    printf("How many number do you want to add? : ");
    scanf("%d", &n);
    for (i = 1; i <= n; i++) {
        printf("Enter number %d: ", i);
        scanf("%d", &num);
        sum = sum + num;
    }
    printf("Sum is: %d", sum);
}



//============================================================================
//============================================================================

File: Python/Basic Maths/Addition.py
# program to add two numbers
num1 = float(input("Enter first number: "))
num2 = float(input("Enter second number: "))
sum = num1 + num2
print("Sum is: ", sum)


#============================================================================
#============================================================================


# program to add n number of integers
sum = 0
totalNumbers = int(input("How many number do you want to add? : "))
for i in range(1, totalNumbers + 1):
    a = int(input(f"Enter number {i}: "))
    sum += a
print("Sum is: ", sum)


#============================================================================
#============================================================================